install/dataloggers.csv

```diff
@@ -1032,6 +1032,7 @@ Nanometrics,Centaur CTR4-6S,9917,Okains Bay,,2023-05-23T01:14:00Z,9999-01-01T00:
 Nanometrics,Centaur CTR4-6S,9918,Rangatira Station,,2023-08-03T11:24:00Z,9999-01-01T00:00:00Z
 Nanometrics,Centaur CTR4-6S,9920,Kaweka Forest,,2023-07-18T03:24:00Z,9999-01-01T00:00:00Z
 Nanometrics,Centaur CTR4-6S,9921,Tory Channel,,2023-06-19T00:07:00Z,9999-01-01T00:00:00Z
+Nanometrics,Centaur CTR4-6S,10922,West Melton 2,,2024-04-04T16:00:00Z,9999-01-01T00:00:00Z
```

Comment:
please update station name to `West Melton Geomagnetic Observatory 2` or `West Melton 2 Geomagnetic Observatory`

Reply:
Done